### PR TITLE
feat(587): Add lastCommitMessage to parsed hook

### PR DESCRIPTION
**Pull request by catto**
## Context

I'd like to implement the skipping builds feature (screwdriver-cd/screwdriver#587)

## Objective

This PR adds lastCommitMessage to the result of `parseHook`.
If this PR is approved, I'll change [around this code](https://github.com/screwdriver-cd/screwdriver/blob/master/plugins/webhooks/index.js#L397) to check if the last commit message contains "[skip ci]". I'll also change other scm plugins.

## References

- issue: screwdriver-cd/screwdriver#587
- webhook spec: https://developer.github.com/v3/activity/events/types/#pushevent

### CODE REVIEW

**Comment by minz1027** [comment]
Probably need to add this for `scm-bitbucket` too so that it's consistent. And update the `readme` for `scm-base`: https://github.com/screwdriver-cd/scm-base#parsehook. Otherwise LGTM

**Comment by catto** [comment]
🆗 I'll add changes for scm-bitbucket and scm-base. Hopefully [scm-gitlab](https://github.com/bdangit/scm-gitlab) also.

File: index.js [hunk 1]
```diff
@@ -678,6 +678,7 @@ class GithubScm extends Scm {
                 sha: hoek.reach(webhookPayload, 'after'),
                 type: 'repo',
                 username: hoek.reach(webhookPayload, 'sender.login'),
+                lastCommitMessage: hoek.reach(webhookPayload, 'head_commit.message') || '',
```
Comment (d2lam) [inline index.js]: You can use `hoek.reach(webhookPayload, 'head_commit.message', { default: '' })`

Comment (catto) [inline index.js]: thanks! I'll use that since next time

**Pull request merged**